src/libs/ReportActionsUtils.ts

```diff
@@ -979,6 +1059,10 @@ function getMemberChangeMessageElements(reportAction: OnyxEntry<ReportAction>):
     const isInviteAction = isInviteMemberAction(reportAction);
     const isLeaveAction = isLeavePolicyAction(reportAction);
 
+    if (!isInviteAction && !isLeaveAction) {
```

Comment:
Previous code used to return an array with some elements even when the action is not an invite or leave action. Is it safe to return [] here?

Reply:
Missed a case for removed action. We can use `isMemberChangeAction` for this.

```
Is it safe to return [] here?
```
Here is the member change message and `targetAccountIDs` only exists on `OriginalMessageChangeLog` type so we can return an empty array which is safe and help us to predict correctly the type of `originalMessage`.